modules/orderly-runner.nix

```
@@ -0,0 +1,56 @@
+{ self, pkgs, config, lib, ... }:
+let
+  inherit (lib) types;
+  imageJson = lib.importJSON ../packages/orderly-runner/image.json;
+  arrayFrom = n: if n == 1 then [ 1 ] else [ n ] ++ arrayFrom (n - 1);
+in
+{
+  options.services.orderly-runner = {
+    enable = lib.mkEnableOption "Orderly runner API";
+    workers = lib.mkOption {
+      description = "Number of workers to spin up";
+      type = types.int;
+    };
+    port = lib.mkOption {
+      type = types.port;
+      default = 8240;
+    };
+  };
+
+  config.services.redis.servers.orderly-runner = {
+    enable = true;
+    port = 6379;
+  };
+
+  config.virtualisation.oci-containers.containers = {
+    orderly-runner-api = rec {
+      imageFile = pkgs.dockerTools.pullImage imageJson;
+      image = "${imageJson.finalImageName}:${imageFile.imageTag}";
```

```suggestion
      image = "${imageJson.finalImageName}:${imageJson.imageTag}";
```

This should work right? That way you can drop the `rec`.

---

yup